client/state/signup/steps/website-content/schema.ts

```diff
@@ -1,140 +0,0 @@
-import type { PageId } from 'calypso/signup/difm/constants';
```

Comment:
Since browser storage is no longer the source of truth, we removed the `withSchemaValidation` function. The schema is also no longer necessary since the entire state is already typed.